sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_call_connection.py

```diff
@@ -64,18 +65,34 @@ def cancel_all_media_operations(
 
     @distributed_trace()
     def play_audio(
-            self,
-            audio_file_uri,  # type: str
-            play_audio_options,  # type: PlayAudioOptions
-            **kwargs  # type: Any
-        ): # type: (...) -> PlayAudioResult
+        self,
+        audio_file_uri,  # type: str
+        play_audio_options,  # type: PlayAudioOptions
+        **kwargs  # type: Any
+    ): # type: (...) -> PlayAudioResult
 
         if not audio_file_uri:
             raise ValueError("audio_file_uri can not be None")
 
         if not play_audio_options:
             raise ValueError("options can not be None")
 
+        try:
+            if not audio_file_uri.lower().startswith('http'):
```

Comment:
let us not add anything to input here. just use some kind of standard url validators in Py